mxnet/training/docker/1.8.0/py3/Dockerfile.cpu

```diff
@@ -131,6 +133,15 @@ RUN ${PIP} install --no-cache --upgrade \
     awscli \
     smclarify
 
+# Install extra packages
+RUN pip install --no-cache-dir -U \
+    bokeh \
+    imageio \
+    opencv-python \
+    plotly \
+    seaborn \
+    shap
```

Comment:
Could you please constrain the packages to the following versions on all dockerfiles:
```
"bokeh>=2.3,<3"
"imageio>=2.9,<3"
"opencv-python>=4.3,<5"
"plotly>=5.1,<6"
"seaborn>=0.11,<1"
"shap>=0.39,<1"
```